app.js

```diff
@@ -21,8 +22,10 @@ const ip = settings.ip || "localhost";
 const appName = settings.app_name || "homework-bet-dev";
 
 app.set('view engine', 'ejs');
-app.use(express.static('public'))
+app.use(express.static('public'));
+app.use(bodyParser.urlencoded( {extended: false} ));
 app.use(bodyParser.json());
+app.use(session({secret: 'bobross'}));
```

Comment:
lets move the session secret out into the settings file so that it's not hardcoded